lib/suspenders/app_builder.rb

```diff
@@ -113,10 +113,6 @@ def set_up_hound
       copy_file "hound.yml", ".hound.yml"
     end
 
-    def configure_newrelic
```

Comment:
The [Skylight docs](https://docs.skylight.io/getting-set-up/) say you don't need to do that:

> For Heroku, setup is as simple as setting the desired ENV vars and pushing your repo. Nothing fancy required!

I'm not experienced enough with Skylight to recommend options, but I'm open to suggestions.